cmd/kubectl-gadget/top/top.go

```diff
@@ -15,16 +15,35 @@
 package top
 
 import (
+	"fmt"
+	"strings"
+	"sync"
+
 	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
 
 	"github.com/spf13/cobra"
 )
 
 // All the gadgets within this package use this global variable, so let's
```

Comment:
`global variableS`.
(Or you can even remove the comment.)

Reply:
I removed the comment.